runtime/test/manifest-test.js

```diff
@@ -461,6 +461,27 @@ describe('manifest', function() {
     let recipe = manifest.recipes[0];
     assert.equal(2, recipe.slots.length);
   });
+  it('multiple consumed slots', async () => {
+    let parseRecipe = async (isRequiredSlotA, isRequiredSlotB, expectedIsResolved) => {
+      let recipe = (await Manifest.parse(`
+        particle SomeParticle in 'some-particle.js'
+          SomeParticle()
+          ${isRequiredSlotA ? 'must ' : ''}consume slotA
+          ${isRequiredSlotB ? 'must ' : ''}consume slotB
+
+        recipe
+          slot 'slota-0' as s0
+          SomeParticle
+            consume slotA as s0
+      `)).recipes[0];
+      recipe.normalize();
+      assert.equal(expectedIsResolved, recipe.isResolved());
+    };
+    await parseRecipe(/* isRequiredSlotA= */ false, /* isRequiredSlotB= */ false, /* expectedIsResolved= */ true);
```

Comment:
nit: if you want self-documenting calls you could put isRequiredSlotA, isRequiredSlotB, expectIsResolved into a dictionary arg?

Reply:
done